templates/display_base.mako

```diff
@@ -101,8 +101,7 @@
         modern_route = modern_route_for_controller(get_controller_name(item))
         item_plural = get_item_plural( item )
         href_to_all_items = h.url_for( controller='/' + modern_route, action='list_published')
-        href_to_user_items = h.url_for( controller='/' + modern_route, action='list_published', xxx=item.user.username)
-        href_to_user_items = href_to_user_items.replace( 'xxx', 'f-username')
+        href_to_user_items = h.url_for( controller='/' + modern_route, action='list_published', **{"f-username": item.user and item.user.username})
```

Comment:
Minor stuff. Do we need to pack and then unpack the `f-username` argument? Maybe just `f-username=item.user and item.user.username`? It's evaluated the same way. (same below)

Reply:
You can't have `-` in argument names:

```
In [2]: dict(f-username=1)
  File "<ipython-input-2-f2b606a950f2>", line 1
    dict(f-username=1)
         ^
SyntaxError: expression cannot contain assignment, perhaps you meant "=="?
```

I think the pack-repack is a little nicer than what was here before.

Reply:
Oh, I see. Sounds good!